src/components/data-exploration/differential-expression-tool/LaunchPathwayAnalysisModal.jsx

```diff
@@ -158,11 +201,41 @@ const LaunchPathwayAnalysisModal = (props) => {
             <b> strongly recommended </b>
             {' '}
             to input the reference list of genes by setting it in &quot;Reference List&quot;
-            then re-run the pathway analysis in the pantherdb page.
+            then re-run the pathway analysis in the PantherDB page.
+            {' '}
+            You can either
             {' '}
-            You can either download reference genes into file or copy reference genes to clipboard.
+            <Button
+              type='link'
+              disabled={gettingBackgroundGenes}
+              loading={gettingBackgroundGenes}
+              style={inlineButtonStyle}
+              onClick={async () => {
+                const genesList = await getBackgroundGenesList();
+                const fileUrl = writeToFileURL(genesList);
+                downloadFromUrl(fileUrl, 'genes_list.txt');
```

Comment:
I would call it perhaps something more specific, like `reference_genes_list.txt` for example. We could get vicky's opinion on this as well.

Reply:
good name :+1: